.github/workflows/auto-merge-bot.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         with:
           node-version: '12'
           check-latest: false
-      - run: cd .github/bot && npm install && npm run build
+      - run: cd .github/bot && npm ci && npm run build
```

Comment:
Provided that we're using tsc would this still be necessary? Or could you just run it directly with node-ts like you mentioned previously?